compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -1868,22 +1868,18 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
             ObligationCauseCode::Coercion { source: _, target } => {
                 err.note(&format!("required by cast to type `{}`", self.ty_to_string(target)));
             }
-            ObligationCauseCode::RepeatVec(suggest_const_in_array_repeat_expressions) => {
-                err.note(
```

Comment:
It looks like you deleted this note by accident.

Reply:
Done